src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/FileVersionInfoTest.Windows.cs

```diff
@@ -127,11 +127,12 @@ public void FileVersionInfo_DifferentFileVersionAndProductVersion()
             });
         }
 
-        private static string GetFileVersionLanguage(uint langid)
+        private static unsafe string GetFileVersionLanguage(uint langid)
         {
-            var lang = new StringBuilder(256);
-            Interop.Kernel32.VerLanguageName(langid, lang, (uint)lang.Capacity);
-            return lang.ToString();
+            const int MaxLength = 256;
+            char* lang = stackalloc char[MaxLength];
+            int charsWritten = Interop.Kernel32.VerLanguageName(langid, lang, MaxLength);
+            return new string(lang, 0, charsWritten);
```

Comment:
I don't love that the tests pull in the same interop src, but I'm not fixing that here, just keeping the tests doing the same thing they were doing.